main/src/com/google/refine/util/HttpClient.java

```diff
@@ -70,7 +76,7 @@ public HttpClient(int delay) {
                 .setDefaultRequestConfig(defaultRequestConfig)
                 .setConnectionManager(connManager)
                 // Default Apache HC retry is 1x @1 sec (or the value in Retry-Header)
-                .setRetryStrategy(new ExponentialBackoffRetryStrategy(3, TimeValue.ofMilliseconds(_delay)))
+                .setRetryStrategy(new ExponentialBackoffRetryStrategy(3, TimeValue.ofMilliseconds(_retryInterval)))
 //                .setRedirectStrategy(new LaxRedirectStrategy()) // TODO: No longer needed since default doesn't exclude POST?
 //               .setConnectionBackoffStrategy(ConnectionBackoffStrategy)
```

Comment:
This TODO can probably be deleted based on the investigation that was done here confirming the comment to be correct.